unity-renderer/Assets/Scripts/MainScripts/DCL/Controllers/ExploreV2/Scripts/MainMenu/ExploreV2Menu/ExploreV2MenuComponentView.cs

```diff
@@ -87,6 +89,8 @@ public override void Awake()
             { ExploreSection.Quest, questSection },
             { ExploreSection.Settings, settingsSection },
         };
+
+        DataStore.i.common.isWorld.OnChange += OnWorldChange;
```

Comment:
We should start moving away from the use of static instances like `DataStore.i` in our classes. In this case we could have some initialization method in the view in order to inject the needed dependencies.

(Maybe it's something that is a bit out of the scope of this PR since there already existed the use of these static instance before your changes, but I wanted to warn it for future improvements)

Reply:
Agree. This refactoring for this legacy code should be done together with removing all `DataStore.i.` appearance there. And there are a lot of them, so I will move it our of scope of this PR